scripts/release.sh

```diff
@@ -15,6 +15,11 @@ function release {
   cabal v1-configure --disable-tests --disable-benchmarks
   cabal v1-sdist
   cabal upload --publish "dist/${NAME}-${VERSION}.tar.gz"
+  if [ "${NAME}" = "dhall-nix" ]
+  then
+    cabal v2-haddock --builddir=docs --haddock-for-hackage --enable-documentation
```

Comment:
Does `cabal v1-haddock` work for this purpose?  The reason I ask is that the `cabal v1-*` commands work with Cabal's Nix support, which I use on the machine I use to release the packages.  If we use the `cabal v2-*` commands then they need to be run inside of a `nix-shell`